codec-http2/src/test/java/io/netty/handler/codec/http2/StreamBufferingEncoderTest.java

```diff
@@ -111,6 +111,11 @@ public void setup() throws Exception {
         when(writer.writeGoAway(any(ChannelHandlerContext.class), anyInt(), anyLong(), any(ByteBuf.class),
                 any(ChannelPromise.class)))
                 .thenAnswer(successAnswer());
+        when(writer.writeHeaders(any(ChannelHandlerContext.class), anyInt(), any(Http2Headers.class),
```

Comment:
This stubbing is fixing a bug in the existing test `receivingGoAwayFailsBufferedStreams`
https://github.com/netty/netty/blob/6724786dcc9fa38ba516dab97a04e2bbc17e81d9/codec-http2/src/test/java/io/netty/handler/codec/http2/StreamBufferingEncoderTest.java#L259

which asserts 9 failure in total. Actually 5 out of 9 are NullPointerExceptions caused by the mock, and only 4 of them are legitimate failures. Here fixes the NullPointerException.